src/localization/bundles/en.json

```diff
@@ -553,6 +553,20 @@
     "server.replace.prepare.failure": "Failed to prepare for replacing server",
     "update.commit.failure": "Failed to commit update changes. {0}",
     "install.progress.failure": "Failed to install SLES OS",
+    "server.retrieve.discovered.servers.error": "Failed to retrieve discovered servers. {0}",
+    "server.retrieve.manual.servers.error": "Failed to retrieve manually added servers. {0}",
+    "server.validating": "Validating Changes...",
+    "server.addserver.prepare": "Preparing Adding Servers in Progress",
+    "server.addserver.prepare.failure": "Failed to prepare for adding servers",
+    "server.addserver.deploy.failure": "Failed to deploy newly added servers",
+    "server.addserver.compute.deploy": "Deploying Compute Servers in Progress",
```

Comment:
I would remove "in Progress" from this string

Reply:
ok will fix.